tests/linker/ios/link sdk/LinkSdkRegressionTest.cs

```diff
@@ -1136,4 +1138,4 @@ public void Principal ()
 			Assert.That (Thread.CurrentPrincipal.Identity.Name, Is.EqualTo ("abc"), "Name");
 		}
 	}
-}
\ No newline at end of file
+}
```

Comment:
your editor does not like ~~stories~~ files that ends well...

Reply:
or you want to make sure you always have the last ~~word~~ char in every file ?

Reply:
Ideally your editor does not change anything you did not touch, including end-of-file.